plugins/externalwrench/include/gazebo/ApplyExternalWrench.hh

```diff
@@ -56,41 +62,29 @@ private:
 public:
     ApplyExternalWrench();
     virtual ~ApplyExternalWrench();
-    std::string retrieveSubscope(gazebo::physics::Link_V& v, std::string  scope);
+    void applyWrenches();
 
-    struct wrench {
-        yarp::sig::Vector force;
-        yarp::sig::Vector torque;
-        double            duration;
-    };
+    bool getCandidateLink(physics::LinkPtr& candidateLink, std::string candidateLinkName);
 
     /// \brief Robot name that will be used to open rpc port
     std::string          robotName;
-    double               timeIni;
-//     yarp::os::Bottle     bufferBottle;
 
+    double               timeIni;
```

Comment:
this variable seems to be unused since the duration check has been moved to ExternalWrench.cc.

Also, I think `#include <yarp/os/Time.h>` could be safely removed.

Reply:
Done